frontend/app/routes/$lang+/_public+/apply+/$id+/age.tsx

```diff
@@ -0,0 +1,288 @@
+          {t('apply:required-label')}
+        </p>
+        <fetcher.Form method="post" aria-describedby="dob-desc form-instructions" noValidate>
+          <input type="hidden" name="_csrf" value={csrfToken} />
+          <div className="mb-6 space-y-4">
+            <h2 className="text-xl font-bold">{t('apply:eligibility.date-of-birth.adult-children-flow.age-heading')}</h2>
+            <DatePickerField
+              id="date-of-birth"
+              names={{
+                day: 'dateOfBirthDay',
+                month: 'dateOfBirthMonth',
+                year: 'dateOfBirthYear',
+              }}
+              defaultValue={defaultState.adultChildrenDateOfBirth ?? ''}
+              legend={t('apply:eligibility.date-of-birth.form-instructions')}
+              errorMessages={{
+                all: fetcher.data?.errors.dateOfBirth?._errors[0],
+                year: fetcher.data?.errors.dateOfBirthYear?._errors[0],
+                month: fetcher.data?.errors.dateOfBirthMonth?._errors[0],
+                day: fetcher.data?.errors.dateOfBirthDay?._errors[0],
+              }}
+              required
+            />
+            <h2 className="text-xl font-bold">{t('apply:eligibility.date-of-birth.adult-children-flow.child-age-heading')}</h2>
+            <InputRadios
+              id="child-under-18"
+              name="allChildrenUnder18"
+              legend={t('apply:eligibility.date-of-birth.adult-children-flow.child-age-instruction')}
+              options={[
+                { value: AllChildrenUnder18Option.Yes, children: t('apply:eligibility.date-of-birth.adult-children-flow.yes'), defaultChecked: defaultState.allChildrenUnder18 === AllChildrenUnder18Option.Yes },
+                { value: AllChildrenUnder18Option.No, children: t('apply:eligibility.date-of-birth.adult-children-flow.no'), defaultChecked: defaultState.allChildrenUnder18 === AllChildrenUnder18Option.No },
+              ]}
+              errorMessage={fetcher.data?.errors.allChildrenUnder18?._errors[0]}
+              required
+            />
+            <Collapsible summary={t('apply:eligibility.date-of-birth.adult-children-flow.collapsible-content-summary')}>
+              <p>{t('apply:eligibility.date-of-birth.adult-children-flow.collapsible-content-detail')}</p>
+            </Collapsible>
+          </div>
+          {editMode ? (
+            <div className="mt-8 flex flex-wrap items-center gap-3">
+              <Button variant="primary" id="continue-button" disabled={isSubmitting} data-gc-analytics-customclick="ESDC-EDSC:CDCP Online Application Form:Save - Date of birth click">
+                {t('apply:eligibility.date-of-birth.save-btn')}
+              </Button>
+              <ButtonLink id="back-button" routeId="$lang+/_public+/apply+/$id+/review-information" params={params} disabled={isSubmitting} data-gc-analytics-customclick="ESDC-EDSC:CDCP Online Application Form:Cancel - Date of birth click">
```

Comment:
The value of `data-gc-analytics-customclick` is duplicated on the `/date-of-birth` route. Should the values be unique per page?

cc @sebastien-comeau

Reply:
Since we're planning to have sub-routes per application types, I assume we will have a specific name based on the current application type. 

Ie.: ESDC-EDSC:CDCP Online Application Form:Myself:Cancel - Date of birth click

It's something we can adjust with the help of Adobe Analytics team.